index.html

```diff
@@ -5045,103 +5045,149 @@ <h2>DID Documents</h2>
       }
     },
     {
-      "id": "did:example:123#8wgRfY3sWmzoeAL-78-oALNvNj67ZlQxd1ss_NX1hZY",
-      "type": "JsonWebKey2020", <span class="comment">// external (property value)</span>
+      "id": "did:example:123#key-7",
+      "type": "JsonWebKey2020",
       "controller": "did:example:123",
       "publicKeyJwk": {
-        "kty": "EC", <span class="comment">// external (property name)</span>
-        "crv": "P-384", <span class="comment">// external (property name)</span>
-        "x": "GnLl6mDti7a2VUIZP5w6pcRX8q5nvEIgB3Q_5RI2p9F_QVsaAlDN7IG68Jn0dS_F", <span class="comment">// external (property name)</span>
-        "y": "jq4QoAHKiIzezDp88s_cxSPXtuXYFliuCGndgU4Qp8l91xzD1spCmFIzQgVjqvcP" <span class="comment">// external (property name)</span>
+        "kty": "RSA", // external (property name)
+        "e": "AQAB", // external (property name)
+        "n": "UkhWaGJGOUZRMTlFVWtKSElBdENGV2hlU1F2djFNRXh1NVJMQ01UNGpWazlraEpLdjhKZU1YV2UzYldIYXRqUHNrZGYyZGxhR2tXNVFqdE9uVUtMNzQybXZyNHRDbGRLUzNVTElhVDFoSkluTUhIeGoyZ2N1Yk82ZUVlZ0FDUTRRU3U5TE8wSC1MTV9MM0RzUkFCQjdRamE4SGVjcHl1c3BXMVR1X0RicXhjU253ZW5kYW13TDUyVjE3ZUtobE80dVh3djJIRmx4dWZGSE0wS21DSnVqSUt5QXhqRF9tM3FfX0lpSFVWSEQxdERJRXZMUGhHOUF6c24zajk1ZC1zYU" // external (property name)
       }
+    }
+  ]
+}        
+      </pre>
+      <pre class="example" title="DID Document with different verification method types">
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/suites/ed25519-2018/v1",
+    "https://w3id.org/security/suites/x25519-2019/v1",
+    "https://w3id.org/security/suites/secp256k1-2019/v1",
+    "https://w3id.org/security/suites/secp256k1recovery-2020/v1",
+    "https://w3id.org/security/suites/jws-2020/v1"
+  ],
+  "verificationMethod": [
+    {
+      "id": "did:example:123#key-0",
+      "type": "Ed25519VerificationKey2018",
+      "controller": "did:example:123",
+      "publicKeyBase58": "3M5RCDjPTWPkKSN3sxUmmMqHbmRPegYP1tjcKyrDbt9J" // external (property name)
     },
     {
-      "id": "did:example:123#NjQ6Y_ZMj6IUK_XkgCDwtKHlNTUTVjEYOWZtxhp1n-E",
-      "type": "JsonWebKey2020", <span class="comment">// external (property value)</span>
+      "id": "did:example:123#key-1",
+      "type": "X25519KeyAgreementKey2019",
+      "controller": "did:example:123",
+      "publicKeyBase58": "FbQWLPRhTH95MCkQUeFYdiSoQt8zMwetqfWoxqPgaq7x" // external (property name)
+    },
+    {
+      "id": "did:example:123#key-2",
+      "type": "EcdsaSecp256k1VerificationKey2019",
+      "controller": "did:example:123",
+      "publicKeyBase58": "ns2aFDq25fEV1NUd3wZ65sgj5QjFW8JCAHdUJfLwfodt" // external (property name)
+    },
+    {
+      "id": "did:example:123#key-3",
+      "type": "EcdsaSecp256k1RecoverySignature2020",
+      "controller": "did:example:123",
+      "ethereumAddress": "0x89a932207c485f85226d86f7cd486a89a24fcc12" // external (property name)
```

Comment:
-1 to adding an `ethereumAddress` example, which is [marked as deprecated](https://w3c.github.io/did-spec-registries/#ethereumaddress), and [has previously raised concerns about brand names](https://github.com/w3c/did-spec-registries/pull/73).

Reply:
@peacekeeper I removed the ethereum address example.